packages/usdk/packages/upstreet-agent/packages/react-agents/classes/discord-manager.ts

```diff
@@ -255,6 +299,11 @@ export class DiscordBot extends EventTarget {
             mentionsRegex: discordMentionRegex,
           });
 
+          conversation.appendCurrentAgentSpecs({
+            mentionId: appId,
```

Comment:
Seems like `mentionId` is specific to Discord, and we will want different `mentionId`s for different clients.

So, maybe this should be an object or an array of "client" or "socials" ids?

```
socialIds: {
  discord: discordId,
  twitter: twitterId,
}
```

or

```
socialIds: [{
  type: 'discord',
  id: appId,
}, {
  type: 'twitter',
  id: ...
}]
```

The latter format would allow for multiple clients of the same type.